auditeval/test.go

```diff
@@ -154,8 +154,8 @@ func getFlagValue(s, flag string) string {
 
 	var flagVal string
 	pttns := []string{
-		flag + `="(.*)"`,
-		flag + `=([^ \n]*)`,
+		flag + `\s*=\s*"(.*)"`,
+		flag + `\s*=([^ \n]*)`,
```

Comment:
I think this will fix this original issue so I'm approving, but I wonder if the second regex would work if there is a space after the equals sign?